### PR TITLE
update the READMEs for 0.16.0

### PR DESCRIPTION
**Pull request by jkwatson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,18 +150,18 @@ This is a **current** feature status list:
...
-| Metrics API                 | v<!--VERSION_UNSTABLE-->0.15.0-alpha<!--/VERSION_UNSTABLE-->  |
-| Metrics SDK                 | v<!--VERSION_UNSTABLE-->0.15.0-alpha<!--/VERSION_UNSTABLE-->  |
-| Prometheus Metrics Exporter | v<!--VERSION_UNSTABLE-->0.15.0-alpha<!--/VERSION_UNSTABLE-->  |
-| OpenTracing Bridge          | v<!--VERSION_UNSTABLE-->0.15.0-alpha<!--/VERSION_UNSTABLE-->  |
-| OpenCensus Bridge           | v<!--VERSION_UNSTABLE-->0.15.0-alpha<!--/VERSION_UNSTABLE-->  |
+| Tracing API                 | v<!--VERSION_STABLE-->0.16.0<!--/VERSION_STABLE-->  |
+| Tracing SDK                 | v<!--VERSION_STABLE-->0.16.0<!--/VERSION_STABLE-->  |
+| Jaeger Trace Exporter       | v<!--VERSION_STABLE-->0.16.0<!--/VERSION_STABLE-->  |
+| Zipkin Trace Exporter       | v<!--VERSION_STABLE-->0.16.0<!--/VERSION_STABLE-->  |
+| Context Propagation         | v<!--VERSION_STABLE-->0.16.0<!--/VERSION_STABLE-->  |
```
Comment (bogdandrutu) [inline README.md:157]: Just `Context`?

**Review by bogdandrutu — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jkwatson) [reply README.md:157]: Sure, I can update it.

Comment (jkwatson) [reply README.md]: also added Baggage, since it hadn't been mentioned in here before

File: README.md [hunk 2]
```diff
@@ -150,18 +150,19 @@ This is a **current** feature status list:
...
-| Zipkin Trace Exporter       | v<!--VERSION_STABLE-->0.15.0<!--/VERSION_STABLE-->  |
-| Context Propagation         | v<!--VERSION_STABLE-->0.15.0<!--/VERSION_STABLE-->  |
-| OTLP Exporter (Spans)       | v<!--VERSION_STABLE-->0.15.0<!--/VERSION_STABLE-->  |
-| OTLP Exporter (Metrics)     | v<!--VERSION_UNSTABLE-->0.15.0-alpha<!--/VERSION_UNSTABLE-->  |
-| Metrics API                 | v<!--VERSION_UNSTABLE-->0.15.0-alpha<!--/VERSION_UNSTABLE-->  |
-| Metrics SDK                 | v<!--VERSION_UNSTABLE-->0.15.0-alpha<!--/VERSION_UNSTABLE-->  |
-| Prometheus Metrics Exporter | v<!--VERSION_UNSTABLE-->0.15.0-alpha<!--/VERSION_UNSTABLE-->  |
-| OpenTracing Bridge          | v<!--VERSION_UNSTABLE-->0.15.0-alpha<!--/VERSION_UNSTABLE-->  |
-| OpenCensus Bridge           | v<!--VERSION_UNSTABLE-->0.15.0-alpha<!--/VERSION_UNSTABLE-->  |
+| Tracing API                 | v<!--VERSION_STABLE-->0.16.0<!--/VERSION_STABLE-->  |
```
Comment (bogdandrutu) [inline README.md:153]: Maybe `Trace API` to reflect the package name?

**Review by bogdandrutu — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (jkwatson) [reply README.md:153]: maybe `Traces API` to match the env vars? :trollface: 

I'm fine with either, really. If you think that's better, happy to make it so.

Comment (bogdandrutu) [reply README.md]: I like more `Trace API`, thanks for doing this :)

**Pull request merged**